### ISSUE TITLE
TRM G2 v2.0 - Inflated BRT Ridership

### ISSUE THREAD

**Issue by ggermainedaniel** [Triangle-Modeling-and-Analytics/TRMG2#438]
The first 2050 EC run using 2.0 produced higher ridership for BRT in some instances by a magnitude of 11 (see below):

BRT Service	2050 MTP	2050 EC	Factor of Increase
New Bern to Downtown Raleigh BRT	3,243	36,649	11
Downtown Raleigh to New Bern BRT	3,498	25,323	7
RTC_Clayton_Garner_LPA	18,293	52,829	3
Clayton_Garner_RTC_LPA	17,513	54,512	3

**Comment by jkolling2** [comment]
PNR and KNR BRT trips seem to be the source of the over-prediction. They are very unrealistically high at all time periods. 

[boardings_and_alightings_by_period.csv](https://github.com/user-attachments/files/18170851/boardings_and_alightings_by_period.csv)
[concatenate_summary.csv](https://github.com/user-attachments/files/18170853/concatenate_summary.csv)

**Comment by dkyleward** [comment]
Definitely a strange result. My first thought is to question just how congested/unrealistic a 2050 future year with only E+C STIP projects is. To be clear, I have no idea what's going on. I'm just wondering at why it would only be this one scenario.

**Comment by jkolling2** [comment]
That was my first thought too, but if it's the kiss-and-ride and park-and-ride trips that are too many, then that logic doesn't make sense because they have to drive too. And they're driving from really far away to the BRT. And not for work trips? 

map images below are from data in _summaries/performance_measures/taz_measures.bin

![2055EC BRT HBW Trips](https://github.com/user-attachments/assets/76f195c8-b5e3-4581-8bb3-a1d93da5474d)
![2055EC BRT Daily Trips](https://github.com/user-attachments/assets/a98e1186-7e6c-40ba-bc02-646f5ec8a56b)
[taz_measures_2055EC2_12_12_24.csv](https://github.com/user-attachments/files/18184564/taz_measures_2055EC2_12_12_24.csv)

**Comment by dkyleward** [comment]
Yes, but congestion in and around the BRT corridors still matters - particularly if it's extreme. For example, there is a flat bonus given to BRT (and rail) that is basically akin to equivalent minutes of travel time. If driving along the BRT corridor is really slow, then this fixed bonus along with (possibly?) being in a reserved row with faster speed, starts to look so appealing that tons of people will choose.

I'm just guessing, of course, but E+C scenarios are routinely very out of whack and can cause odd results. It might make sense to do a test scenario with the SE data scaled back a bit and see what happens.

For what it's worth, I stopped doing future-year E+C scenarios as a baseline shortly after leaving CAMPO. It creates an impossible scenario as the baseline and throws off analysis. Instead, I started doing something like 10+ years of SE data growth as the E+C. Prioritize projects based on that, and then run a few variant alts into 2050.

**Comment by jkolling2** [comment]
Ok - I'll try running a 2020 E+C scenario to see what happens. Thanks, Kyle!

**Comment by Flatiron87** [comment]
The congested VMT in the adopted 2050 mtp using v1.3.2 is significantly more than the congested VMT in the 2055EC using v2 (see image). The mtp2050 BRT is 79568-on (2148 drive access). The 2055EC BRT is 173,608-on (107338 drive access). It would seem that if the congested VMT was pushing more riders to BRT, that we would have seen it in v1.3.2.
![image](https://github.com/user-attachments/assets/fb222516-21b7-4fca-8f6a-b509d8500048)

**Comment by yzhang57** [comment]
@dkyleward 
Kyle,

DCHC has no BRT line in the (E+C) scenario. However, the model reports a significant number of BRT trips. By "significant," we mean that the number of BRT trips exceeds what would be expected from a BRT line of/within the DCHC area, and it spreads in almost all TAZs.

If your schedule permits, we would greatly appreciate your assistance in resolving this issue, as well as other concerns outlined in the related issue report. We are working under tight deadline constraints and may need to explore alternative resources if a resolution cannot be reached in a timely manner.

Thanks for your attention,

Yanping 
> Yes, but congestion in and around the BRT corridors still matters - particularly if it's extreme. For example, there is a flat bonus given to BRT (and rail) that is basically akin to equivalent minutes of travel time. If driving along the BRT corridor is really slow, then this fixed bonus along with (possibly?) being in a reserved row with faster speed, starts to look so appealing that tons of people will choose.
> 
> I'm just guessing, of course, but E+C scenarios are routinely very out of whack and can cause odd results. It might make sense to do a test scenario with the SE data scaled back a bit and see what happens.
> 
> For what it's worth, I stopped doing future-year E+C scenarios as a baseline shortly after leaving CAMPO. It creates an impossible scenario as the baseline and throws off analysis. Instead, I started doing something like 10+ years of SE data growth as the E+C. Prioritize projects based on that, and then run a few variant alts into 2050.

**Comment by yzhang57** [comment]
Morning Kyle,
Thanks for the prompt reply. we cannot give up the (E+C) scenario. Even giving up it , we will have other 1-2 alternatives without BRT lines in the DCHC area and most likely TRMg2 will still generate unreasonable and significant BRT trips in the DCHC area based on what Mike Bruff's test with more highway projects. Additionally, TRMg2 generates considerably lower travel time in comparison to the 2050 MTP. Taking travel from Durham to Raleigh as one example, our planner pointed out the PM travel time 20-30 minutes lower in the 2055 (E+C) scenario. 
Best,
Yanping

**Comment by yzhang57** [comment]
Hi Si,

Thanks for the suggestion. Yes. I can do it. Please understand we are working under tight deadline constraints. So we cannot wait for the monthly meeting and may other Caliper experts if Kyle is busy. Thanks.

Yanping

**Comment by ajyoshizumi** [comment]
To address the issue, the following amendments were made:

- A penalty was added to v0 households using pnr_brt within the HBNW constant/coefficient tables.
- Scaling factors were derived from HBW tables to address inflated BRT ridership numbers stemming from problematic pnr_brt and knr_brt constants within the HBNW constant/coefficient tables.
- Erroneous pnr_brt and knr_brt trips occurring from the w_hb_o_all trip purpose were removed.
- we implemented a travel time ratio threshold that eliminates knr_brt trips where 60% or more of the trip was drive time + in-vehicle travel time.
